tests/unit/test_row_noise.py

```diff
@@ -74,15 +80,25 @@ def test_do_not_respond(mocker, dummy_data):
     )
 
     # Test that noising affects expected proportion with expected types
-    assert np.isclose(
-        1 - len(noised_data1) / len(my_dummy_data), config[Keys.ROW_PROBABILITY], rtol=0.02
+    fuzzy_checker.fuzzy_assert_proportion(
+        name="test_do_not_respond",
+        observed_numerator=len(my_dummy_data) - len(noised_data1),
+        observed_denominator=len(my_dummy_data),
+        target_proportion=config[Keys.ROW_PROBABILITY],
+        name_additional=f"noised_data1",
     )
     assert set(noised_data1.columns) == set(my_dummy_data.columns)
     assert (noised_data1.dtypes == my_dummy_data.dtypes).all()
 
     # Check ACS data is scaled properly due to oversampling
     expected_noise = 0.5 + config[Keys.ROW_PROBABILITY] / 2
-    assert np.isclose(1 - len(noised_data2) / len(my_dummy_data), expected_noise, rtol=0.02)
+    fuzzy_checker.fuzzy_assert_proportion(
+        name="test_do_not_respond",
```

Comment:
does the name actually matter? note that this name is the same as the one above on line 83

Reply:
It does not. It is used in the output file you can check. There is a name_additional parameter that you can pass to give additional information to debug from the outputs if need be.

Reply:
We should avoid having different fuzzy asserts with the same name and name_additional, since we wouldn't be able to distinguish them in the diagnostic file.

Reply:
Yes - but isn't name additional for this purpose? If you want to have multiple assertions in the same test but be able to distinguish them?

Reply:
Yep!

Reply:
Ok so I did implement correctly.

Reply:
Confused myself for a second.